src/main/kotlin/org/onflow/flow/sdk/FlowAccessApi.kt

```diff
@@ -3,55 +3,60 @@ package org.onflow.flow.sdk
 import com.google.protobuf.ByteString
 
 interface FlowAccessApi {
-    fun ping()
+    sealed class FlowResult<out T> {
+        data class Success<out T>(val data: T) : FlowResult<T>()
+        data class Error(val message: String, val throwable: Throwable? = null) : FlowResult<Nothing>()
```

Comment:
```suggestion
    sealed class AccessApiCallResponse<out T> {
        data class Success<out T>(val data: T) : AccessApiCallResponse<T>()
        data class Error(val message: String, val throwable: Throwable? = null) : AccessApiCallResponse<Nothing>()
```
Wanted to make the naming here a little more definitive and keep the _Flow_ prefix on types here to indicate canonical data from the chain itself. This entity is more a response wrapping those canonical data types

Reply:
Agree - will propagate this throughout. Thanks!